roles/generate-jenkins/templates/ISSUE_TEMPLATE.feature.j2

```diff
@@ -1,32 +1,44 @@
----
+# Based on the issue template
 name: Feature request
-about: Suggest an idea for this project
-
----
+description: Suggest an idea for this project
+title: "[FEAT] <title>"
+labels: [enhancement]
+body:
 {% if project_deprecation_status %}
-# DEPRECATION NOTICE
+  - type: markdown
+    attributes:
+      value: |
+        # DEPRECATION NOTICE
 
-This image is deprecated. We will not offer support for this image and it will not be updated.
-{% if project_deprecation_message is defined %}{{ project_deprecation_message }}{% endif %}
+        This image is deprecated. We will not offer support for this image and it will not be updated.
+{% if project_deprecation_message is defined %}
 
+        {{ project_deprecation_message | indent(8) | trim }}
+{% endif %}
+{% else %}
+  - type: checkboxes
+    attributes:
+      label: Is there an existing feature request for this?
+      description: Please search to see if an feature request already exists.
+      options:
+        - label: I have searched the existing issues
+          required: true
```

Comment:
I think the wording on the label vs the option label are contradicting. It's two separate questions.
1. Is there an existing feature request for this?
2. I have searched the existing issues
Where # 1 should be no and # 2 should be yes.

Maybe something like this instead?
```suggestion
  - type: checkboxes
    attributes:
      label: Is this a new feature request?
      description: Please search to see if a feature request already exists.
      options:
        - label: I have searched the existing issues
          required: true
```

Reply:
Tbh, I just copy-pasted this from the issue one, replacing the words, but this makes sense.